### ISSUE TITLE
Create a sample app with Vue.js

### ISSUE THREAD

**Issue by vrajdesai78** [Huddle01/Sample-Apps-Monorepo#1]
## Objective
Create a sample application using [Vue.js](https://vuejs.org/) and integrate our [Vanilla JS SDK](https://docs.huddle01.com/docs/Javascript) to demonstrate the following features:

- Lobby screen to join a room
- Enable Video/Audio functionality
- Grid to show other peers' viewport

## Picking up this issue
If you'd like to work on this, please comment "picking this up" below, and I'll assign the issue to you.

## Questions
- If you need assistance or clarification, feel free to comment below.
- You can also ask your doubts 24x7 in our [Discord channel](https://discord.com/invite/huddle01-890224574761926756).
- Check your ranking after finishing this task on the [Leaderboard](https://huddle01.notion.site/Huddle01-Hacktoberfest-Leaderboard-34c909db00eb4828aab3f1da8502a6fa?pvs=4).

## Resources
- Familiarize yourself with our [Vanilla JS SDK](https://docs.huddle01.com/docs/Javascript).
- Explore the [Vue.js documentation](https://vuejs.org/guide/introduction.html) to understand how to build applications using Vue.js.

**Good Luck! 🍀**

**Comment by Adtrex** [comment]
I would like to learn more about this, I am actually confused with the goal here

**Comment by scarlettverse** [comment]
Hey @Adtrex,

Could you please clarify what you would like more details on specifically?

**Comment by Adtrex** [comment]
I meant is it just testing the following features listed above

**Comment by Adtrex** [comment]
I mean 
-creating a test application for the features

**Comment by scarlettverse** [comment]
Yes @Adtrex , you should create a sample app with the listed features and use the specified tech stack.

**Comment by Nupoor10** [comment]
picking this up

**Comment by scarlettverse** [comment]
Hey @Nupoor10 , how's it going with this issue?
